### ISSUE TITLE
Correct the conflicting language of summaries in verbose output

### ISSUE THREAD

**Issue by XLTechie** [restic/restic#4565]
Output of `restic version`
--------------------------
restic 0.16.2 compiled with go1.21.3 on linux/386

What should restic do differently? Which functionality do you think we should add?
----------------------------------------------------------------------------------
When performing a backup for the first time, e.g.:
```
restic backup --cleanup-cache --verbose -p /root/backup.p --compression max \
home root opt \
var/{smb,www,local}
```

The following output is generated:
```
Files:       1085848 new,     0 changed,     0 unmodified
Dirs:        16015 new,     0 changed,     0 unmodified
```
Reading that first line, zero files were "changed".
But, at the same time, zero files were also "unmodified". Since "unmodified" means "not modified", and since 0 files were not modified, it seems apparent that every file was modified.
*Another way of saying "0 UNmodified", is "1085848 modified". Or: none were changed, and all were changed.*
Thus, in effect, the above line is saying:
```
Files:       1085848 new,     0 changed, 1085848 modified
```

How can no files be changed, but also every file be modified?
Same for directories.

What are you trying to do? What problem would this solve?
---------------------------------------------------------
While I know the important number here is "0 changed", having "0 unmodified" (meaning "every file modified") at the same time, is unnecessarily confusing. It's also a double negative--"none not modified".

**At the very least, this should say "0 changed, 1085848 modified". That still wouldn't make sense, but at least it wouldn't involve a double negative.**

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------
It's a great backup utility, and has all the features I need for handling Linux server and workstation backups.
I even introduced it to a colleague who maintains Windows systems, who finds it quite satisfying.

**Comment by MichaelEischer** [comment]
> How can no files be changed, but also every file be modified?

It means that all files are new. Each file in a snapshot is either "new" (file did not exist in parent snapshot), "changed" (a file with the same name existed, but has different content/metadata) or "unmodified" (same as in the parent snapshot). That means that the opposite of `unmodified` is `new+changed`.

I'm aware that `changed` and `unmodified` don't convey all nuances of how they are used in the backup output. But I also don't see an alternative that does.

> However regardless, if the point of unmodified is supposed to be to give a contrast to "changed", it should use the same terminology.
> E.G. "100 changed, 500 unchanged".

Renaming `unmodified` to `unchanged` is even more confusing. `1085848 new,     0 changed,     0 unchanged` is rather strange as apparently a category and its inverse don't cover all files, which really feels wrong. In that respect, the different words for `changed` and `unmodified` serve the purpose of showing that both aren't exact opposites of each other.

In short, the naming of `changed` and `unmodified` isn't optimal, but `changed` and `unchanged` isn't the solution either. That is, I think we should keep the current names unless there's some other, better suggestion.

What do you think about adding a new subsection to https://restic.readthedocs.io/en/stable/040_backup.html which explains the output of the backup command?

> "n/a changed, n/a unchanged"

Omitting both fields is probably clearer. Although, I'd prefer to not complicate the summary output format.
